3-async-await/solution.js

```diff
@@ -19,17 +19,83 @@ Example:
 8. add any needed adjustment to solution() function
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
+const prices = require("./prices.js");
+const products = require("./products.js");
 
-function solution() {
-    // YOUR SOLUTION GOES HERE
+async function solution() {
+    // is this a bad practice? declaring but no initalization of variables that will eventually have a value in all logic branches?
```

Comment:
It really depends on the context of your method, but in this case is totally valid since you're going to use each variable.

In some cases it is not a good idea to initialize all of the variables due to internal represents so space reserved to do something.

For example if you're not going to use a variable because the method has an error abruptly it can be an example of not assigning a variable.